tests/common/helpers/portchannel_to_vlan.py

```diff
@@ -400,7 +400,7 @@ def setup_po2vlan(duthosts, ptfhost, rand_one_dut_hostname, rand_selected_dut, p
                 pytest.skip("OS Version of fanout is older than 202205, unsupported")
             asic_type = fanouthost.facts['asic_type']
             platform = fanouthost.facts["platform"]
-            if not (asic_type in ["broadcom"] or platform in
+            if not (asic_type in ["broadcom", "mellanox"] or platform in
```

Comment:
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/acl/test_acl_outer_vlan.py#L656-L674

This check is identical to test_acl_outer_vlan.py. Could we update that test as well?

Reply:
Hi @bingwang-ms , it was used to skiped by https://github.com/sonic-net/sonic-mgmt/pull/11309, but I found that it actually could run pass at mellanox fanout switch, so I created this PR.

Reply:
> https://github.com/sonic-net/sonic-mgmt/blob/master/tests/acl/test_acl_outer_vlan.py#L656-L674
> 
> This check is identical to test_acl_outer_vlan.py. Could we update that test as well?
Hi @ganglyu , I will test and feed back.

Reply:
Hi @ganglyu , test_acl_outer_vlan.py could be run over mellanox sonic fanout, I had updated the skip condition both.